website/layout/AutodocsLayout.js

```diff
@@ -140,15 +143,42 @@ var ComponentDoc = React.createClass({
         {(composes || []).map((name) =>
           this.renderCompose(name)
         )}
-        {Object.keys(props).sort().map((name) =>
+        {Object.keys(props)
+          .sort((a, b) => {
+            a = props[a];
+            b = props[b];
+
+            if (a.platforms && !b.platforms) {
+              return 1;
+            }
+            if (b.platforms && !a.platforms) {
+              return -1;
+            }
+            return a.name < b.name;
+          })
+          .map((name) =>
```

Comment:
Whoops, just noticed the props are not sorted alphabetically (although this code looks good).

https://facebook.github.io/react-native/docs/image.html#content